notification/eventType.go

```diff
@@ -0,0 +1,18 @@
+package notification
```

Comment:
filename should not be camel-cased. This file could me merged with notification.go.